.gitignore

```diff
@@ -1,2 +0,0 @@
-# Documentation will be periodically generated and placed in the gh-pages branch
-docs/html
```

Comment:
Please don't remove the gitignore. We don't need docs/html in master (as it tends to clutter commits like this)